tests/utils/esxcli/cmd.go

```diff
@@ -47,3 +56,55 @@ func KillVM(esxHostIP, worldID string) bool {
 	}
 	return true
 }
+
+// RetrieveVMNameFromIP util retrieves VM  name from passed VM IP
+//esxcli vm.info -vm.ip=10.20.104.62 -json | jq -r .VirtualMachines[].Name
+func RetrieveVMNameFromIP(ip string) string {
+	log.Printf("Finding VM name from IP Address [%s]\n", ip)
```

Comment:
For all these utility functions, lets not put a log for each function. Once we have debug logs for tests then we can add these. For now please leave the printf() calls but leave them commented out. If some test is misbehaving then the developer can enable that log and test. Lets not have these logs turned on.